src/odata-to-abstract-sql.ts

```diff
@@ -1536,14 +1536,19 @@ export class OData2AbstractSQL {
 			}
 			if (resource.fields.some((f) => f.computed != null)) {
 				const computedFieldQuery = new Query();
-				computedFieldQuery.select = resource.fields.map((field) =>
-					this.AliasSelectField(
-						resource,
-						sqlNameToODataName(field.fieldName),
-						field.computed,
-						field.fieldName,
-					),
-				);
+				computedFieldQuery.select = [
+					['Field', '*'],
+					...resource.fields
+						.filter(({ computed }) => computed != null)
```

Comment:
NIT: Perhaps we could run this `.filter()` outside the `if (...some())` and just check the length of the result in the if statement.

Reply:
See: https://github.com/balena-io-modules/odata-to-abstract-sql/pull/86